build/authors.py

```diff
@@ -0,0 +1,17 @@
+"""
+Process author table
+"""
+
+import pandas
+
+
+def get_author_info(author_file):
+    """
+    Load the author table and return a dict with author information such as
+    the number of authors.
+    """
+    author_df = pandas.read_table(author_file)
+
+    author_info = dict()
```

Comment:
Let's use a `collections.OrderedDict` here since we want the `stats.json` to have a deterministic ordering.